packages/manager/modules/octavia-load-balancer/src/translations/Messages_fr_FR.json

```
@@ -1,3 +1,4 @@
 {
-  "octavia_load_balancer": "Load Balancer"
+  "octavia_load_balancer": "Load Balancer",
+  "octavia_load_balancer_global_error": "L'erreur suivante est arrivée : <br> {{message}}<br> request_id: {{requestId}}"
```

```suggestion
  "octavia_load_balancer_global_error": "L'erreur suivante est survenue : <br> {{message}}<br> request_id: {{requestId}}"
```
In french, `une erreur est arrivée` is a bit weird